### PR TITLE
Db creation

### PR DESCRIPTION
**Pull request by pankajholariyacb**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -425,6 +436,12 @@ variable "grn_secret_name" {
   default     = "grn-db-secrets"
 }
 
+variable "commandcenter_secret_name" {
+  description = "The the secret name to use"
+  type        = string
+  default     = "commandcenter-db-secrets"
```
Comment (itsmesuniljacob) [inline variables.tf:442]: better to have it as `command-center-db-secrets`

File: variables.tf [hunk 2]
```diff
@@ -425,6 +436,12 @@ variable "grn_secret_name" {
   default     = "grn-db-secrets"
 }
 
+variable "commandcenter_secret_name" {
```
Comment (itsmesuniljacob) [inline variables.tf:439]: `command_center_secret_name` @pankajholariyacb

File: postgres.tf [hunk 3]
```diff
@@ -81,6 +81,12 @@ resource "random_string" "grn_user_password" {
   override_special = "@#%&*()-_=+[]{}<>:?"
 }
 
+resource "random_string" "commandcenter_user_password" {
```
Comment (itsmesuniljacob) [inline postgres.tf:84]: `command_center_user_password`

File: postgres.tf [hunk 4]
```diff
@@ -191,6 +197,11 @@ module "inventory_google_postgres" {
       charset   = "UTF8"
       collation = "en_US.UTF8"
     },  
+    {
+      name      = var.commandcenter_database
```
Comment (itsmesuniljacob) [inline postgres.tf:201]: `command_center_db_name`

File: postgres.tf [hunk 5]
```diff
@@ -241,6 +252,10 @@ module "inventory_google_postgres" {
       name     = "grn-user"
       password = random_string.grn_user_password.result
     },
+    {
+      name     = "commandcenter-user"
```
Comment (itsmesuniljacob) [inline postgres.tf:256]: `command-center-user`

File: variables.tf (hunk 1 above)
Comment (pankajholariyacb) [reply variables.tf]: Added seperation between command and center words

File: variables.tf (hunk 2 above)
Comment (pankajholariyacb) [reply variables.tf]: resolved

File: postgres.tf (hunk 3 above)
Comment (pankajholariyacb) [reply postgres.tf]: resolved

File: postgres.tf (hunk 4 above)
Comment (pankajholariyacb) [reply postgres.tf]: resolved

File: postgres.tf (hunk 5 above)
Comment (pankajholariyacb) [reply postgres.tf]: resolved

**Review by Sunil-Jacob — APPROVED** [review]
LGTM

**Pull request merged**